app/modules.go

```diff
@@ -284,6 +284,14 @@ func orderInitBlockers() []string {
 		paramstypes.ModuleName,
 		upgradetypes.ModuleName,
 		vestingtypes.ModuleName,
+		// The globalfee module should ideally be initialized before the genutil module in theory:
```

Comment:
Something like this makes it clearer to me:

The ordering of modules globalfee and gentx are dependent on the global fee value.

If global fee > 0, then gentx needs to pay the fee, ...
If global fee is empty, then gentx can be initialized before globalfee

To avoid ambiguous behaviour, we initialize the globalfee module after genutil, ensuring that the global
min fee is empty when gentx is called.

For more details, please refer to the following link: https://github.com/cosmos/gaia/issues/2489

Reply:
it is rather the global fee should init before gentx. we did a tricky hack this way.